tests/test_file_handler.py

```
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+import gc
+from pathlib import Path
+import threading
+
+from femtologging import FemtoFileHandler
+
+
+def test_file_handler_writes_to_file(tmp_path: Path) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)